tests/CppUTestExt/IEEE754PluginTest_c.c

```diff
@@ -25,6 +25,10 @@
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include <CppUTest/CppUTestConfig.h>
+
+#ifdef CPPUTEST_HAVE_FENV
```

Comment:
Would you mind putting this in your PR (or I can submit a separate one):
```c++
void set_overflow_c(void)
{
    f = 1e38f;
    f *= f;
}

void set_underflow_c(void)
{
    f = 1e-38f;
    f *= f;
}
```
I couldn't find any good substitute for sqrt(-1), so math.h will have to stay.